openprescribing/frontend/management/commands/measure_sql/vw__median_price_per_unit.sql

```diff
@@ -3,19 +3,20 @@
 --
 --     ./manage.py create_bq_measure_views
 
-WITH
-  prices_per_unit AS (
+WITH prices_per_unit AS (
   SELECT
     month AS date,
     bnf_code,
     IEEE_DIVIDE(net_cost,quantity) AS price_per_unit
   FROM
-    {project}.{hscic}.normalised_prescribing_standard)
+    {project}.{hscic}.normalised_prescribing_standard
+  WHERE quantity > 0
```

Comment:
Can we add a reference to https://github.com/ebmdatalab/openprescribing/issues/1373?